lib/verk/queue_stats.ex

```diff
@@ -24,25 +25,29 @@ defmodule Verk.QueueStats do
 
   @doc false
   def init(_) do
-    :ets.new(@table, [:ordered_set, :named_table, read_concurrency: true, keypos: 1])
+    QueueStatsCounters.init
     { :ok, nil }
   end
 
   @doc false
   def handle_event(%Verk.Events.JobStarted{ job: job }, state) do
-    :ets.update_counter(@table, job.queue, { 2, 1 }, default_tuple(job.queue))
+    QueueStatsCounters.job_event(:started, job.queue)
     { :ok, state }
   end
 
   def handle_event(%Verk.Events.JobFinished{ job: job }, state) do
-    :ets.update_counter(@table, job.queue, [{ 3, 1 }, { 2, -1 }], default_tuple(job.queue))
+    QueueStatsCounters.job_event(:finished, job.queue)
     { :ok, state }
   end
 
   def handle_event(%Verk.Events.JobFailed{ job: job }, state) do
-    :ets.update_counter(@table, job.queue, [{ 4, 1 }, { 2, -1 }], default_tuple(job.queue))
+    QueueStatsCounters.job_event(:failed, job.queue)
     { :ok, state }
   end
 
-  defp default_tuple(queue), do: { queue, 0, 0, 0 }
+  @doc false
+  def handle_info(:persist_stats, state) do
+    QueueStatsCounters.persist
```

Comment:
I changed the `persist` function to return the return value of the last write to Redis. It will return then, the total count of processed and failed jobs. How should we log this?